tests/tests.PrintDiamond.cpp

```diff
@@ -7,3 +7,8 @@ TEST(testHiker, answer_test)
 {
 	EXPECT_EQ(42, answer());
 }
+
+TEST(testDiamond, print_diamond)
```

Comment:
Please add unit tests to your project. They should, in fact, normally come before the code.